src/dlgAboutDialog.h

```diff
@@ -42,6 +50,7 @@ class dlgAboutDialog : public QDialog, public Ui::about_dialog
     void setLicenseTab(const QString& htmlHead) const;
     void setThirdPartyTab(const QString& htmlHead) const;
     void setSupportersTab(const QString &htmlHead);
+    QString createMakerHTML(const aboutMaker) const;
```

Comment:
How about using a reference instead - to save the argument copy? I.e. change to:
```
    QString createMakerHTML(const aboutMaker&) const;
```

Reply:
Committed